tests/Unish/UnishTestCase.php

```diff
@@ -276,12 +281,58 @@ function setUpDrupal($num_sites = 1, $install = FALSE, $version_string = UNISH_D
     return self::$sites;
   }
 
-  function fetchInstallDrupal($env = 'dev', $install = FALSE, $version_string = UNISH_DRUPAL_MAJOR_VERSION, $profile = NULL) {
-    $options = array();
-    $root = $this->webroot();
-    $site = "$root/sites/$env";
+  function setUpSingleSiteDrupal($env = 'dev', $install = FALSE, $version_string = UNISH_DRUPAL_MAJOR_VERSION, $profile = NULL) {
```

Comment:
I like the way that alias creation, db-url generation, and site placement is encapsulated in the UnishTestCase class.  I'd have to do that in each test if we just used archive restore.  There's only one test that uses single site right now, but single site is A Thing, so I am guessing that we might use it in some other tests at some point.  I'll see if I can refactor the code in UnishTestCase, or otherwise make it easier to duplicate single sites in the tests.

Oh, the other thing about archive-restore is that we'd then have one single site "dev" with a sites folder named "dev", and another single site named "stage" with a sites folder named "dev".  Again, considering the use-case of using this in more tests, I thought that might get confusing, so the current code uses "default" for the sites folder when making single sites.